services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -97,12 +103,15 @@
     @Getter(AccessLevel.PROTECTED)
     private final RetryTemplate retryTemplate;
 
+    @Setter(onMethod_ = {@Autowired})
+    @Getter(AccessLevel.PROTECTED)
+    private PaymentResponseUtil paymentResponseUtil;
```

Comment:
```suggestion
    @Getter(AccessLevel.PROTECTED)
    private final PaymentResponseUtil paymentResponseUtil;
```
This service isn't being used, so it's essentially a new integration. With that in mind, we don't have to worry about backward-compatibility. Lets add the util to the constructor!